### PR TITLE
[bugfix] Remove softmax layer during finetuning

### PR DESCRIPTION
**Pull request by guillaumebrg**

### CODE REVIEW

**Review by antoinemrcr — COMMENTED** [review]
I can be ok with that, but would not it be better if we had a parameter for the logisticResgression to include or not the softmax at initialization ?

**Review by antoinemrcr — APPROVED** [review]
LGTM
